src/mjolnir/landmarks.cc

```diff
@@ -352,8 +354,10 @@ void FindLandmarkEdges(const boost::property_tree::ptree& pt,
       // get landmarks in the tile
       midgard::AABB2<PointLL> bbox = baldr::TileHierarchy::GetGraphIdBoundingBox(tileset[i]);
 
-      std::vector<Landmark> landmarks =
-          db.get_landmarks_by_bbox(bbox.minx(), bbox.miny(), bbox.maxx(), bbox.maxy());
+      std::vector<Landmark> landmarks = db.get_landmarks_by_bbox(bbox.minx() - kLandmarkQuerryBuffer,
+                                                                 bbox.miny() - kLandmarkQuerryBuffer,
+                                                                 bbox.maxx() + kLandmarkQuerryBuffer,
+                                                                 bbox.maxy() + kLandmarkQuerryBuffer);
```

Comment:
on arm, apparently when locations lie directly on the bounding box boundary (in this case the equator and the prime meridian) `st_covers` was not returning the landmarks. you might think that its just floating point precision in the actual landmarks but if that were the case then we'd iterate over more 1x1 degree tiles and get the landmarks, however we dont, our test still ends up doing only 1 1x1 query. so i copped out and added a 1 meter buffer

Reply:
I wonder, where would one report spatialite bugs😅it’s so awkward to not have it on some git platform. Every time I think, that’s it, now it’s over, a new version comes out from somewhere..